gradle/publish-mpp-root-module-in-platform.gradle

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+
+/** Publish the platform JAR and POM so that consumers who depend on this module and can't read Gradle module
+ metadata can still get the platform artifact and transitive dependencies from the POM: */
+project.ext.publishPlatformArtifactsInRootModule = { platformPublication ->
```

Comment:
Please leave a comment to https://youtrack.jetbrains.com/issue/KMM-237#focus=streamItem-27-4115233.0-0 here

Reply:
Done!